modules/ROOT/pages/configure-tls-context-rtf-lb.adoc

```diff
@@ -0,0 +1,58 @@
+= Configure a TLS Context for Anypoint Runtime Fabric
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+All inbound traffic entering Anypoint Runtime Fabric is encrypted using TLS. When enabling inbound traffic, you must provide a valid TLS certificate. When you configure inbound traffic for Runtime Fabric, you can configure the TLS server by importing a Privacy-Enhanced Mail (PEM) or Java Keystore (JKS) file, or by importing a TLS context from the secrets manager (advanced use case).
+
+Before you can import the TLS Context from the secrets manager, you must add the TLS context. 
+
+== Add the TLS Context to a Secret Group
+
+Adding a TLS context to your secret group involves supplying a name, target, security information, version, keystore and, optionally, truststore. 
+
+. In the *Secret Groups* list view, select the secret group to which you want to add a TLS context. 
```

Comment:
Should "secret group" be "secrets group"? I initially changed it above, but now I am not sure.
To me, "secret group" sounds like a group that is secret, not a group that contains secrets.
I'll stop marking.